sdk/src/main/java/com/mapbox/maps/NativeMapImpl.kt

```diff
@@ -534,6 +534,14 @@ internal class NativeMapImpl(val map: MapInterface) :
     return map.setStyleGeoJSONSourceData(sourceId, data)
   }
 
+  override fun setStyleGeoJSONSourceData(
```

Comment:
@pozdnyakov it seems that introducing the new method broke the `url` usecase for the GeoJsonSource. If I call `setStyleGeoJSONSourceData` (doesn't matter new or old method) with  `GeoJSONSourceData` created with `url` of the source - it's not loaded for some reason

Reply:
@yunikkk thanks fir pointing it out, here is the fix https://github.com/mapbox/mapbox-gl-native-internal/pull/4357

Reply:
@pozdnyakov testing it, thanks.

Good thing on the testing side - it was actually a UI test that initially had caught the issue, the generated `GeoJsonSourceTest.urlTest` cc @mapbox/maps-android .

Reply:
> Good thing on the testing side - it was actually a UI test that initially had caught the issue

that's good 👍 . For any changes related to style (source,layer etc). we should run `run-style-test`, which can catch issues like these.

Reply:
@ank27 passes https://app.circleci.com/pipelines/github/mapbox/mapbox-maps-android/7800/workflows/a1ca55c9-2704-4041-950a-64347eb5cf49/jobs/67225!